styles/styles.css

```diff
@@ -334,6 +334,76 @@ main .section.gray-gradient {
   );
 }
 
+.angle-background-top,
+.angle-background-top-s,
+.angle-background-top-l {
+  background: var(--white);
+}
+
+.section .angle-background-top::after,
+.section .angle-background-top-s::after {
+  background: var(--gray-gradient-light);
+  clip-path: polygon(0 100%, 100% 5vw, 100% 0, 0 100%);
```

Comment:
Are we sure this is always based on the literal view width, and not the container's width?

Reply:
The `5vw` is to provide a vertical length that preserves the aspect ratio essentially so the angle of the line/slant doesn't change when the viewport changes sizes.